src/main/java/pl/pop/interview/master/question/QuestionController.java

```diff
@@ -26,6 +26,15 @@ public List<QuestionDTO> getAllQuestions() {
         return questionFacade.getAllQuestions();
     }
 
+    @GetMapping("/{id}/answers")
+    public List<QuestionDTO> getQuestionsForPractitioner(@PathVariable("id") Long id, @RequestParam(required = false) Boolean answered) {
```

Comment:
Tak na dobrą sprawę, to tutaj się powinno zwracać "ResponseEntity<Page < QuestionDTO > >"

Reply:
OK - kontroler teraz zwraca ResponseEntity<Page<QuestionDTO>>